addon/styles/components/email.scss

```diff
@@ -0,0 +1,122 @@
+.cs-component-email {
+  // local variables
+  --cs-component-email-border-radius: var(--cs-input-border-radius);
+  --cs-component-email-label: var(--cs-input-label);
+  --cs-component-email-validation: var(--cs-input-valid);
+  --cs-component-email-validation-icon: url('/@cardstack/ui-components/images/check-icon.svg');
+  --cs-component-email-validation-icon-size: 1em;
+
+  position: relative;
+  padding-bottom: 3em;
+  font-size: var(--cs-input-font-size);
+}
+
+.cs-component-email--input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 60px;
+  margin: 0;
+  padding-left: 15px;
+  padding-top: 20px;
+
+  border: 1px solid var(--cs-input-border);
+  border-radius: var(--cs-component-email-border-radius);
+  background-color: var(--cs-input-background);
+  color: var(--cs-input-copy);
+  font-family: inherit;
+  font-size: inherit;
+  line-height: 22px;
+
+  overflow-x: hidden; /* Hack to make "rows" attribute apply in Firefox. */
+  -webkit-transition: padding-top 0.2s ease, margin-top 0.2s ease;
+  transition: padding-top 0.2s ease, margin-top 0.2s ease;
+}
```

Comment:
A lot of this styling is also done in the TextField component. Should we think about DRYing this up?